doc/user/config.toml

```diff
@@ -52,14 +52,28 @@ identifier = 'redpanda'
 weight = 20
 
 [[menu.main]]
-identifier = "network-security"
-name = "Network security"
+identifier = "webhooks"
+name = "Webhooks"
 parent = 'ingest-data'
-weight= 30
+weight = 30
 
 [[menu.main]]
-identifier = "webhooks"
-name = "Webhooks"
+identifier = "eventbridge"
+name = "Amazon EventBridge"
+parent = 'webhooks'
+weight = 5
+url = "/sql/create-source/webhook/#connecting-with-amazon-eventbridge"
+
+[[menu.main]]
+identifier = "segment"
+name = "Segment"
+parent = 'webhooks'
+weight = 15
+url = "/sql/create-source/webhook/#connecting-with-segment"
```

Comment:
I think you can just remove this section to solve all your linting problems! I don't have all the context, but it seems redundant unless you are planning to get rid of the segment guide
![image](https://github.com/MaterializeInc/materialize/assets/11527560/ec2e86eb-79a3-4784-bbc8-56240cdf899a)